ibm_mq/datadog_checks/ibm_mq/config.py

```diff
@@ -0,0 +1,49 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.config import is_affirmative
+
+# compatability layer for agents under 6.6.0
```

Comment:
Let's not do this and just pin base if need be

Reply:
This was done because we needed to install it on agent versions where this change was not active yet. In fact, we were testing this before 6.6 went out. As we will still be asking people who might not have upgraded to custom install this, I am not going to make this change